nmos-testing-facade.py

```diff
@@ -55,10 +55,16 @@ def index():
 
             # POST to test suite to confirm answer available
             valid, response = do_request('POST', form['response_url'], json=answer_json)
+
+            if not valid:
+                return response
```

Comment:
This function seems to have several possible return types (response object, tuple of string and status code, or just a string). Do all of these get interpreted by the caller as intended?

Reply:
Yes, it's a good point, and I will check. I have a feeling that success/errors may be handled inconsistently.

Reply:
Responses and error handling in the Testing Facade have now been generally improved.